mitodl/micromasters
# fix broken link to MITx Online Dashboard

#### Pre-Flight checklist


#### What are the relevant tickets?
fixes #5134 

#### What's this PR do?
fix broken link to MITx Online Dashboard

#### How should this be manually tested?
Will be tested on RC, just check the build is passing

NOTE: After performing all the steps mentioned in the PR description, I found out that we have set up the environment variable MITXONLINE_BASE_URL to https://courses-qa.mitxonline.mit.edu/. But that should be set to https://rc.mitxonline.mit.edu/ and for the dashboard link, I have created the PR.
